### ISSUE TITLE
Unhandled Exception after loading map and TEdit 0.0.0.0.0.0

### ISSUE THREAD

**Issue by Sl3ith** [TEdit/Terraria-Map-Editor#738]
I tried the latest Beta release with Terraria 1.3, the software starts (even if it asks me to check the latest version because it wasn't able to verify which version I was using)but when I try to open a random map but it doesn't load it, and if I try to select one of the tools I get the "Unhandled exception" bug and the editor close.

This is the log (I had to upload it here because it's quite long)
[TEditLog.txt](https://github.com/TEdit/Terraria-Map-Editor/files/108533/TEditLog.txt)

**Comment by helpmeoutbro** [comment]
same

**Comment by BinaryConstruct** [comment]
This is a very strange issue. I have downloaded your log, and Microsoft (https://msdn.microsoft.com/en-us/library/windows/desktop/bb509553%28v=vs.85%29.aspx) seems to think that your graphics card was removed from your computer, which I doubt is the actual case.

I'll keep looking to see if I can find any other resolutions to this issue.

http://www.monogame.net/ Monogame hasn't released a stable build in quite a while. Hopefully the next update from this dependency will solve some of these low level GPU issues.

Thank you for your patience, unfortunately at this I do not have any suggestions other than a graphics driver and/or graphics card update.
